src/lib/EventInfo.svelte

```diff
@@ -20,29 +23,110 @@
 	$: dateFormatter = event.allDay ? englishDateOnlyFormatter : englishDateTimeFormatter;
 
 	$: dateRange = dateFormatter.formatRange(event.start!, event.end!);
+
+	type Attachment = {
+		fileId: string;
+		fileUrl: string;
+		iconLink: string;
+		mimeType: string;
+		title: string;
+	};
+
+	type ExtendedProps = {
+		attachments: Attachment[];
+		description?: string;
+		location?: string;
+	};
+
+	$: extendedProps = event.extendedProps as ExtendedProps;
 </script>
 
-<p>{event.title}</p>
+<div class="event-detail-grid">
+	<h3 class="header">{event.title}</h3>
 
-<p>{dateRange}</p>
+	<svg
+		xmlns="http://www.w3.org/2000/svg"
```

Comment:
We should probably make a component for SVG icons to avoid code duplication

Reply:
Fixed in [186a4be](https://github.com/kendoclub-at-umich/website/pull/96/commits/186a4bed6d1512a2b311d1d6c27b95cc92b6c2c8)